### ISSUE TITLE
Rss is Working 

### ISSUE THREAD

**Issue by arthurcgoes** [w3c/feedvalidator#38]
www.saude.al.gov.br/feed

**Comment by dontcallmedom** [comment]
While this bug report lacks details, running the [feedvalidator on the above URL](https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fwww.saude.al.gov.br%2Ffeed) shows a problem with the configuration of the server when it comes to its SSL certificates, with an [incomplete certificate chain](https://www.ssllabs.com/ssltest/analyze.html?d=www.saude.al.gov.br&latest) - see #30 where a similar problem was described.

In other words, this is not a problem with the validator, so closing the issue.
